xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -4073,8 +4083,8 @@ Writes exactly 8 bytes to the beginning of the span.
       <Docs>
         <param name="destination">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Writes a <see cref="T:System.UIntPtr" /> into a span of bytes, as little endian.</summary>
+        <returns>If the span is too small to contain the value, return <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="false" /> if the span is too small to contain the value; otherwise, <see langword="true" />.</returns>
```